### PR TITLE
Use complete string for the block id to hotplug/hotunplug

### PR DESCRIPTION
**Pull request by abdhaleegit**
Added support to pass full string of block id like 'memory234',
it saves few steps of the user to find the memory blocks and call
this functions to hotplug/unplug

Signed-off-by: Abdul Haleem <abdhalee@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by abdhaleegit** [comment]
with reference to #2216

File: avocado/utils/memory.py [hunk 1]
```diff
@@ -37,17 +37,32 @@ class MemError(Exception):
...
+def get_blk_string(block):
+    """
+    Format the given block id to string
+
+    :param block: memory block id or block string.
+    :type string: like 198 or memory198
+    :return: returns string memory198 if id 198 is given
+    :rtype: string
+    """
+    block = [block]
```
Comment (clebergnu) [inline avocado/utils/memory.py]: Why do you turn this string into a list (with the string on it) if you always access the string itself? I'm really confused here.

File: avocado/utils/memory.py [hunk 2]
```diff
@@ -37,17 +37,32 @@ class MemError(Exception):
...
+    if not block[0].startswith("memory"):
+        return "memory%s" % block[0]
+    return block[0]
+
+
```
Comment (clebergnu) [inline avocado/utils/memory.py]: I mean:
```python
if not block.startswith("memory"):
    return "memory%s" % block
return block
```
Does the same thing.

File: avocado/utils/memory.py (hunk 1 above)
Comment (abdhaleegit) [reply avocado/utils/memory.py]: Yes, you are right. converting to list is unnecessary here. It was a added for different reason in old PR, so not valid now. I will fix this

File: avocado/utils/memory.py (hunk 2 above)
Comment (abdhaleegit) [reply avocado/utils/memory.py]: Yes, it does the same thing. fixing it

**Pull request merged**